src/tests/Common/dirs.proj

```diff
@@ -18,9 +18,10 @@
     </ItemGroup>
 
     <ItemGroup>
-      <AllProjects Include="$(TestRoot)**\*.csproj" Exclude="@(DisabledProjects)" />
+      <!-- <AllProjects Include="$(TestRoot)**\*.csproj" Exclude="@(DisabledProjects)" />
       <AllProjects Include="$(TestRoot)**\*.fsproj" Exclude="@(DisabledProjects)" />
-      <AllProjects Include="$(TestRoot)**\*.ilproj" Exclude="@(DisabledProjects)" />
+      <AllProjects Include="$(TestRoot)**\*.ilproj" Exclude="@(DisabledProjects)" /> -->
+      <AllProjects Include="$(TestRoot)JIT\Methodical\divrem\div\i4div_cs_do.csproj" />
```

Comment:
Where does this come from?

Reply:
This is a hack for faster development. It will be reverted before merged.